http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -53,6 +53,21 @@ public static FeignClientFactory standard() {
     }
 
     /**
+     * Provides a {@link FeignClientFactory} compatible with jackson 2.4.
```

Comment:
want to leave a comment saying that users of this constructor need to force their jackson dependency to 2.4 ?